Test/FloatTorchTensorUnitTest.cs

```diff
@@ -0,0 +1,368 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+using System.Numerics.Tensors;
+using Torch.SNT;
+
+namespace Torch.SNT
+{
+    [TestClass]
+    public class FloatTorchTensorUnitTest
```

Comment:
Should we auto-generate the unit tests for all types?

Reply:
Yes! Let me do it.